### ISSUE TITLE
switch not doing what it should do & wiki typo

### ISSUE THREAD

**Issue by Dotolox** [TomSchimansky/CustomTkinter#276]
Hey, 
I really like this package. However, I'm facing huge problems when trying to integrate the switch object.

First of all I guess in the wiki there are some typos, when it comes to the switch:
wiki:
`switch_var = tkinter.StringVar("on")`
should be:
`switch_var = tkinter.StringVar(value="on")`

wiki:
```
def switch_event():
    print("switch toggled, current value:", switch_1.get())
```
should be:
```
def switch_event():
    print("switch toggled, current value:", switch_var.get())
```

However, when integrating that in the right way, I still can't use the object, the way I would like to.
This piece of code creates an switched off button:
```
                switch_var = StringVar(value="off")

                def switch_event():
                    print(switch_var.get())

                switch_1 = customtkinter.CTkSwitch(master=app, text="Auto Captcha", command=switch_event, variable=switch_var, onvalue="on", offvalue="off", width=40, height=20)
                switch_1.place(
                    x=290.0,
                    y=370.0
                )
```
Nevertheless, when clicking on it, it forwards the value 'off' again and ongoing from then it delivers the value inverted. So when turned off its on and when on it returns off.
Setting `onvalue='off'` and `offvalue='on'` removes this inverted effect for me, however at the beginning(when the button is drawn the first time) it's still wrong(button turend on, but returning off).

**Comment by TomSchimansky** [comment]
Wiki should be fixed now.

**Comment by TomSchimansky** [comment]
This was indeed a bug with the CTkSwitch, the variable was updated after the command function was called. Thanks for reporting this. It's now fixed with version 4.5.6.
